### PR TITLE
fix regex

### PR DESCRIPTION
**Pull request by benoxoft**

### CODE REVIEW

**Comment by mikefaille** [comment]
LGTM

File: clean.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 def func(columns):
     # @[a-z0-9_]+) : remove twitter user beginning with prefix '@'
     # (https?://(?:[-\w.]|(?:%[\da-fA-F]{2}))+) : remove urls starting with 'http(s)'
```
Comment (mikefaille) [inline clean.py]: Just update this comment please

**Review by mikefaille — CHANGES_REQUESTED** [review]

**Comment by benoxoft** [comment]
@mikefaille fixed

**Review by ttauveron — APPROVED** [review]
LGTM

**Review by mikefaille — APPROVED** [review]
lgtm

**Pull request merged**